From 7192da8f33ba3a04aa7fd77459600362d178e783 Mon Sep 17 00:00:00 2001
From: Jun Li <r65092@freescale.com>
Date: Fri, 27 Jun 2008 14:02:44 +0800
Subject: [PATCH] ENGR00082083 I.MX35 Change the LAN9217 driver to 16bit mode

On 3stack board, LAN9217 connectted with CS5 by 16Bits bus width.
Changed LAN9217 to work in 16Bit mode.

Signed-off-by: Jun Li <r65092@freescale.com>
---
 drivers/net/smsc911x.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/net/smsc911x.h b/drivers/net/smsc911x.h
index 9bbcc0f..d2c6d33 100644
--- a/drivers/net/smsc911x.h
+++ b/drivers/net/smsc911x.h
@@ -24,6 +24,9 @@
 #ifdef CONFIG_MACH_MX37_3DS
 #define SMSC_CAN_USE_SPI	1
 #define SMSC_CAN_USE_32BIT	0
+#elif defined(CONFIG_MACH_MX31_3DS) || defined(CONFIG_MACH_MX35_3DS)
+#define SMSC_CAN_USE_32BIT      0
+#define SMSC_CAN_USE_SPI        0
 #else
 #define SMSC_CAN_USE_32BIT	1
 #define SMSC_CAN_USE_SPI	0
-- 
1.5.4.4

